Views/backend/connect/controller/main.js

```diff
@@ -1096,20 +1157,21 @@ Ext.define('Shopware.apps.Connect.controller.Main', {
                 if (!operation) {
                     return;
                 }
-
                 if (!operation.success) {
                     me.createGrowlMessage(title, operation.message, true);
                 } else {
-                    if (operation.sourceIds.length > 1000) {
+                    if (operation.count > 1000) {
 
                         Ext.create('Shopware.apps.Connect.view.export.product.manyProductsDialog', {
-                            sourceIds: operation.sourceIds
+                            exportAll: true,
```

Comment:
Imho these bool switches are also bad practice
maybe we can find a cleaner solution here